src/Form/FormControlSet.jsx

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
+/** @jsx jsx */
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { jsx, useTheme } from '@emotion/react';
+import { formControlSetStyle } from './style';
 
 const FormControlSet = ({
   as,
   className,
   isInline,
   children,
   ...props
-}) => React.createElement(as, {
-  className: classNames(
-    className,
-    {
-      'pgn__form-control-set': !isInline,
-      'pgn__form-control-set-inline': isInline,
-    },
-  ),
-  ...props,
-}, children);
+}) => {
+  const style = formControlSetStyle(useTheme());
+  return jsx(as, {
+    className: classNames(className, 'pgn__form-control-set'),
+    css: isInline ? style.inline : style.base,
```

Comment:
the css prop can be a function that accepts theme. It's probably better to use that form instead of grabbing the theme with `useTheme`

```suggestion
    css: (theme) => {
      const style = formControlSetStyle(theme);
      if (isInline) 
        return style.inline;
      }
      return style.base;
    },
```